.github/workflows/tests.yml

```diff
@@ -13,55 +13,28 @@ on:
       - '**.md'
 
 jobs:
-  test:
-    name: Run tests
-
-    strategy:
-      fail-fast: false
-      matrix:
-        node_version:
-          - 14
-          - 16
+  verify:
+    name: Verify
 
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-
-      - name: Setup
-        uses: actions/setup-node@v2.5.1
         with:
-          node-version: ${{matrix.node_version}}
+          fetch-depth: 0
```

Comment:
Setting `fetch-depth: 0` makes it so `git describe --tags` works in the `make docker` step.